curriculum/dictionaries/chinese/comments.js

```diff
@@ -6,435 +6,435 @@
-    text: 'comment placeholder (needs translation)'
+    text: '将它们合并起来'
   },
   {
     id: '5j2l88',
-    text: 'comment placeholder (needs translation)'
+    text: '我们创建三个浏览器窗口'
   },
   {
     id: 'e843r9',
-    text: 'comment placeholder (needs translation)'
+    text: '我们现在打开一个新的选项卡'
   },
   {
     id: '5fvehh',
-    text: 'comment placeholder (needs translation)'
+    text: '在 myLocalScope 之外，未定义 myVar'
   },
   {
     id: 'qn720a',
-    text: 'comment placeholder (needs translation)'
+    text: '现在，在 console.log() 之前添加 console.clear()，清除浏览器控制台的内容，通过测试。'
   },
   {
     id: 'j86mef',
-    text: 'comment placeholder (needs translation)'
+    text: '现在执行打开选项卡，关闭选项卡和其他操作'
   },
   {
     id: 'mk7rvy',
-    text: 'comment placeholder (needs translation)'
+    text: '现在移除 console log 这一行，通过测试'
   },
   {
     id: 'n7vm1s',
-    text: 'comment placeholder (needs translation)'
+    text: '只修改这一行上面的代码'
   },
   {
     id: 'cvh4x7',
-    text: 'comment placeholder (needs translation)'
+    text: '只修改这一行下面的代码'
   },
   {
     id: 'lvmnm7',
-    text: 'comment placeholder (needs translation)'
+    text: '打开一个新的选项卡，显示猫的图片'
   },
   {
     id: 'avpx79',
-    text: 'comment placeholder (needs translation)'
+    text: '打开你的浏览器控制台。'
   },
   {
     id: '0b5ps6',
-    text: 'comment placeholder (needs translation)'
+    text: '画布边线和图表之间的 padding'
   },
   {
     id: 'uemoej',
-    text: 'comment placeholder (needs translation)'
+    text: '将 n 个 0 推入当前行以创建列'
   },
   {
     id: 'lm86nf',
-    text: 'comment placeholder (needs translation)'
+    text: '将当前行（已有 n 个 0）推送到数组'
   },
   {
     id: 'qscelx',
-    text: 'comment placeholder (needs translation)'
+    text: '可从 Redux 对象获得 Redux 方法'
   },
   {
     id: 'atqiig',
-    text: 'comment placeholder (needs translation)'
+    text: '在这一行下面渲染一个输入框（input），按钮（button）和列表（ul）'
   },
   {
     id: 'yq81wg',
-    text: 'comment placeholder (needs translation)'
+    text: '在这一行下面渲染 Provider'
   },
   {
     id: 'kxio9j',
-    text: 'comment placeholder (needs translation)'
+    text: 'responseFromServer 设置为 false，表示从服务器获得无效响应'
   },
   {
     id: 'alh6pw',
-    text: 'comment placeholder (needs translation)'
+    text: 'responseFromServer 设置为 true，表示从服务器获得有效响应'
   },
   {
     id: '1cfidd',
-    text: 'comment placeholder (needs translation)'
+    text: 'responseFromServer 表示从服务器获得一个响应'
   },
   {
     id: '96tntk',
-    text: 'comment placeholder (needs translation)'
+    text: '返回 30'
   },
   {
     id: '58a5g7',
-    text: 'comment placeholder (needs translation)'
+    text: '运行并检查控制台'
   },
   {
     id: '71bus9',
-    text: 'comment placeholder (needs translation)'
+    text: '运行测试，查看两个控制台的差异。'
   },
   {
     id: '7wp46n',
-    text: 'comment placeholder (needs translation)'
+    text: '设置为华氏刻度'
   },
   {
     id: 'oefvg5',
-    text: 'comment placeholder (needs translation)'
+    text: '设置'
   },
   {
     id: 'mnt4d3',
-    text: 'comment placeholder (needs translation)'
+    text: '应该显示 'carrot''
```

Comment:
Because the string contains `'`, the entire string needs to be wrapped in double quotes.
```suggestion
    text: "应该显示 'carrot'"
```